CHANGES.md

```diff
@@ -1,5 +1,17 @@
 # pyfakefs Release Notes
-The release versions are PyPi releases.
+The released versions correspond to PyPi releases.
+
+## Preview of future Version 4.0
+
+_Note:_ pyfakefs 4.0 is planned for release at the end of 2019 or the beginning
+of 2020. As pyfakefs 4.0 is a major release, we are giving you advance notice of
+the proposed changes so you can be ready.
+
+  * pyfakefs 4.0 drops support for Python 2.7. If you still need
+    Python 2.7, you can still use the latest pyfakefs 3.x version. 
+
+  * pyfakefs 4.0 version will also remove the old-style API, which was
```

Comment:
The old-style API needs to be described. This seems like a major breaking change.

Reply:
Yes, I'm still not completely sure if we shall remove it - I have no idea how frequently it is still used. What do you think?
By "describing" the old API you mean giving a few examples, and/or pointing to the old documentation?

Actually I could check a few packages on GitHub that depend on pyfakefs to get some impression of the API usage. There are 330 repos listed...